### PR TITLE
Extend the SBO identifiers and add isa_xxx functions

### PR DESCRIPTION
**Pull request by stelmo**

### CODE REVIEW

File: src/utils/looks_like.jl [hunk 1]
```diff
@@ -140,10 +140,14 @@ forwarded to [`looks_like_extracellular_metabolite`](@ref).
...
-@_is_reaction_fn "pseudo" Identifiers.PSEUDOREACTIONS
-@_is_reaction_fn "metabolic" Identifiers.METABOLIC_REACTIONS
-@_is_reaction_fn "spontaneous" Identifiers.SPONTANEOUS_REACTIONS
+@_is_sbo_reaction_fn "exchange" Identifiers.EXCHANGE_REACTIONS
+@_is_sbo_reaction_fn "transport" Identifiers.TRANSPORT_REACTIONS
+@_is_sbo_reaction_fn "biomass" Identifiers.BIOMASS_REACTIONS
+@_is_sbo_reaction_fn "atp_maintenance" Identifiers.ATP_MAINTENANCE_REACTIONS
+@_is_sbo_reaction_fn "pseudo" Identifiers.PSEUDO_REACTIONS
+@_is_sbo_reaction_fn "metabolic" Identifiers.METABOLIC_REACTIONS
+@_is_sbo_reaction_fn "spontaneous" Identifiers.SPONTANEOUS_REACTIONS
```
Comment (exaexa) [inline src/utils/looks_like.jl:149]: I'd vote for using symbols here right away, as in:
```suggestion
@_is_sbo_reaction_fn :spontaneous Identifiers.SPONTANEOUS_REACTIONS
```

not a big deal but prevents weird errors like leaving a space in the string (which doesn't translate very well to function names :D )

**Review by exaexa — APPROVED** [review]
lovely, consider the 2 comments (both minor nitpicking)

File: src/utils/looks_like.jl (hunk 1 above)
Comment (stelmo) [reply src/utils/looks_like.jl:149]: Cool, I will do this, but in FBCMTs.

**Comment by stelmo** [comment]
Note, all of this sbo stuff will also go to FBCMTs, so after this merge, expect a PR cutting it out :)

**Pull request merged**